Sources/Private/Experimental/Layers/ShapeItemLayer.swift

```diff
@@ -59,32 +59,51 @@ final class ShapeItemLayer: CALayer {
     let parentGroup: Group?
   }
 
-  override func layoutSublayers() {
-    super.layoutSublayers()
+  override func setupAnimations(context: LayerAnimationContext) {
+    super.setupAnimations(context: context)
 
-    for sublayer in sublayerConfiguration?.layers ?? [] {
-      sublayer.fillBoundsOfSuperlayer()
```

Comment:
Adding a `BaseAnimationLayer` conformance lets us remove this custom layout code